### ISSUE TITLE
[Adaptive Card iOS - Expense Report.Json]: Voice Over does not convey the state for the 'Reject' button present under 'ExpenseReport.json' card.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8442]
### Target Platforms

iOS

### SDK Version

1.0 (2.4.0-beta.23.1.8.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 16.3.1
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.23.1.8.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.5" control and select it.
5. Navigate to 'Scenario' control and activate it.
6. Navigate to ' ExpencesReport.Jsons' and activate it.
7. The Card will open.
8. Navigate to Passenger name and seat number.
9. Observe voice over issue.

**Actual Result:**
Voice Over does not convey the state for the 'Reject' button present under 'ExpenseReport.json' card. Voiceover is only announcing as, "Reject button".

**Expected Result:**
Voice over should convey the collapsed state for the 'Reject' button present under 'ExpenseReport.json' card. Voiceover should announce as, "Reject button collapsed".

**User Experience:**
It will impact the screen reader user if the screen reader does not convey the state for the any control on the screen.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/229745166-535cf853-1b6d-44fd-a0af-903c4e3aad21.mov

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
This issue is now tracked with below URL, hence closing this issue.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/49
